### PR TITLE
ADD -disableCustomNotifications CLI

### PR DESCRIPTION
**Pull request by fgalan**
Implements #2135

### CODE REVIEW

File: test/functionalTest/cases/2135_disable_custom_notifications_cli/disble_custom_notifications_cli.test [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+#
+# This is a a variant of case 2015 notification_templates_many_notifications.test using
+# the disableCustomNotifications CLI
+#
+# 01. Create E1/T1
+# 02. Create E1/T2
+# 03. Create E1/T3
+# 04. Create regular subscription for E1 -> 3 entities in one notification
+# 05. Create same subscription but with 'httpExtended' instead of 'http' -> 3 entities in one notification
+# 06. Create extended subscription for E1 -> 3 entities in one notification
```
Comment (kzangeli) [inline test/functionalTest/cases/2135_disable_custom_notifications_cli/disble_custom_notifications_cli.test]: I miss some kind of clarification here, about why the broker behaves the way it behaves.
Something like:

    -> 3 entities in one notification (as the broker treats httpExtended as http when started with -disableCustomNotifications)

Comment (fgalan) [inline test/functionalTest/cases/2135_disable_custom_notifications_cli/disble_custom_notifications_cli.test]: Too long... note that disableCustomNotifications is mentioned in preamble at L35. That should be enough.

Thus, I propose NTC here.

Comment (kzangeli) [inline test/functionalTest/cases/2135_disable_custom_notifications_cli/disble_custom_notifications_cli.test]: ok

NTC then

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**